src/pages/KadraVykhovnykiv/UpdateKadraForm.tsx

```diff
@@ -90,7 +90,7 @@ const UpdateKadraForm: React.FC<FormUpdateKadraProps> = (props: any) => {
                 label="Дата вручення"
                 labelCol={{ span: 24 }}
                 name="dateOfGranting"
-                initialValue={moment.utc(record.dateOfGranting).local()}
+                initialValue={moment.utc(record.dateOfGranting)}
```

Comment:
do you need this conversion to the UTC here? The date will become -2 hours, won`t it?